### ISSUE TITLE
Provide a way to run several fuzzers instance in parallel

### ISSUE THREAD

**Issue by jvoisin** [fuzzitdev/pythonfuzz#16]
It would be nice to have an option like libfuzzer's [`-jobs`]( https://releases.llvm.org/5.0.0/docs/LibFuzzer.html ) one, to be able to run several instances of the same fuzzer in parallel without having a use `tmux` :)
